GameOfLifeApp/Services/Serializer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Newtonsoft.Json;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Saves JSON data to a file.    
+    /// </summary>
+    public class Serializer : ISerializer
+    {
+        // Declaring a private string. Get the complete path of the current working directory. Using CurrentDirectory property of Environment class.
+        private string _fileName = $"{Environment.CurrentDirectory}\\GameData.json";
+
+        /// <summary>
+        /// Creates a StreamWriter and adds some text to the writer using StreamWriter.
+        /// </summary>
+        /// <param name="gameLogic">Saves the data of the object.</param>
+        public void SaveData(IGameField field)
+        {
+            using (StreamWriter writer = File.CreateText(_fileName))
+            {
+                string output = JsonConvert.SerializeObject(field);
+                writer.Write(output);
+            }
+        }
+
+        public void Load()
```

Comment:
Load in general should return the game field object so you can use it to play the game not just show once the last iteration of it.

Reply:
what is the point of load method if it only shows saved game, it should take the loaded field and continue the game if I want to use a file, otherwise there is no point in saving if you can't restore the game and continue it.

Reply:
Done.